heron/scheduler-core/tests/java/com/twitter/heron/scheduler/dryrun/Constants.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.scheduler.dryrun;
+
+public final class Constants {
+
+  public static final String TEST_DATA_PATH =
+      "/__main__/heron/scheduler-core/tests/java/com/twitter/heron/scheduler/dryrun/testdata";
```

Comment:
I don't know why we wrote our own java_tests target, but if we still need that, we can easily add resources to it. It's just a proxy to the bazel java_test method which takes resources.

Ideally we should write tests to load files using relative paths based on the classpath instead of absolute paths with leaks from the build system included (e.g., __main__). Those files should live in a resources/ dir instead of being mixed into the java source.

If we can't achieve that with the java_test 'resources' attribute, we can do so with the 'data' attribute but we should first try patching our java_tests impl. It's fine to do this test correctly and then open an issue to refactor others to use the proper pattern.